docs/source/user_guide/installation/index.rst

```diff
@@ -45,6 +45,10 @@ Dependencies
 ------------
 .. _install.required_dependencies:
 
+.. warning::
+
+    The list of dependencies below might be outdated. Please refer to the pyproject.toml file to find all dependencies.
```

Comment:
```suggestion
    The list of dependencies below might be outdated. Please refer to the ``pyproject.toml`` file to find all dependencies.
```

Reply:
I have updated it
![image](https://github.com/user-attachments/assets/669334e3-246e-4e73-a481-a0639524b2dd)